app/views/athletes/index.html.erb

```diff
@@ -1 +1,9 @@
+<h1>Athletes</h1>
+
+<% Athlete.all.each do |athlete| %>
+  <athlete id="athlete">
```

Comment:
Having this inside the loop will create a new "athlete" block for each the athletes. 
```
<ul id="athlete">
<% Athlete.all.each do |athlete| %>
 <li><%= link_to athlete.name, athlete_path(athlete)%></li>
<% end %>
</ul>
```

Might be a better way for managing the hole list.